api/test/pact/pact-tests/rdCommon/getCaseFlagRefData.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { expect } from 'chai';
+import * as config from 'config';
+import * as sinon from 'sinon';
+import { mockReq, mockRes } from 'sinon-express-mock';
+import { PactTestSetup } from '../settings/provider.mock';
+import { getRdCommonDataAPIOverrides } from '../utils/configOverride';
+import { requireReloaded } from '../utils/moduleUtil';
+
+const { Matchers } = require('@pact-foundation/pact');
+const { somethingLike } = Matchers;
+const pactSetUp = new PactTestSetup({ provider: 'referenceData_commonData', port: 8000 });
+
+const serviceId = 'IA';
+
+describe('RD get case flag ref data', async () => {
+  const RESPONSE_BODY = {
+    name: somethingLike('caseflag example'),
+    hearingRelevant: somethingLike(true),
+    flagComment: somethingLike(false),
+    flagCode: somethingLike('xflag'),
+    isParent: somethingLike(false),
+    Path: somethingLike('/flag1'),
+    childFlags: somethingLike([])
+  };
+
+  describe('get case flag ref data', () => {
+    const sandbox: sinon.SinonSandbox = sinon.createSandbox();
+    let next;
+
+    beforeEach(() => {
+      next = sandbox.spy();
+    });
+
+    before(async () => {
+      await pactSetUp.provider.setup();
+      const interaction = {
+        state: 'get case flag ref data',
+        uponReceiving: 'a service id',
+        withRequest: {
+          method: 'GET',
+          path: `/refdata/commondata/caseflags/service-id=${serviceId}`,
```

Comment:
That's not present in the getCaseFlagRefData function so would cause pact test failure unless the function itself was edited with said query. Are you happy for me to leave a comment making note of this?

Reply:
Oh, OK... in that case, yes - just leave a comment on that. Thanks